subsys/logging/log_backend_std.h

```diff
@@ -8,6 +8,7 @@
 
 #include <logging/log_msg.h>
 #include <logging/log_output.h>
+#include <irq.h>
```

Comment:
What was the reason for that @nordic-krch ?

Reply:
irq locking is used in static inline function here. Since includes has been cleaned in other log headers, it was missing here.